cf-agent/verify_users_pam.c

```diff
@@ -1403,6 +1458,42 @@ static bool DoModifyUser (const char *puser, const User *u, const struct passwd
     return true;
 }
 
+#ifdef __FreeBSD__
+struct passwd *
+fgetpwent(FILE *stream)
+{
+    struct passwd *pw;
+    char *line;
+    size_t linecap;
+    ssize_t linelen;
+    int pwd_scanflag;
+
+    pw = NULL;
+    line = NULL;
+    linecap = 0;
+    pwd_scanflag = 0;
+
+    if (stream == NULL)
+            return NULL;
```

Comment:
This function is very similar to the other one above. Please:

* Move the argument `if` check to the top.
* Combine variable declarations and initializations.
* Add curly braces to `if`s.